.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
```

Comment:
@crazy-max (sorry for the ping); should this have the same rules for "push" and "tag" as the other ones? https://github.com/distribution/distribution/blob/fee8c686bb60b21fda9dcb1c6b1306e79000e395/.github/workflows/validate.yml#L3-L10